tests/core/vm/contract_fixture.py

```diff
@@ -35,51 +33,96 @@
-                                            ['mstore', 64, 118167469832824678325960435],
-                                            ['clamp_nonzero', ['create2', ['mload', 32], ['mload', 32], 85, 11]],  # noqa: E501
-                                            ['sstore', 0, ['add', ['mload', 32], 1]],
-                                            'stop'],
-                                    0]]]  # noqa: E128
-
-
-# compiled byte code
-CREATE2_contract_bytecode = '0x610039566000546020526a61beef6020526002603ef3604052600b6055602051602051fe8061002957600080fd5b50600160205101600055005b61000461003903610004600039610004610039036000f3'  # noqa: E501
-
-
-# address where this contract will be deployed
-CREATE2_contract_address = generate_CREATE2_contract_address(
-    b'',
-    decode_hex(CREATE2_contract_bytecode)
-)
-
-
-# PAYGAS Contract
-PAYGAS_contract_lll_code = ['seq',
-                                ['return',  # noqa: E127
-                                    0,
-                                    ['lll',
-                                        ['seq',
-                                            ['paygas', ['calldataload', 64]],
-                                            # Transfer value
-                                            ['mstore', 128, 1461501637330902918203684832716283019655932542976],  # noqa: E501
-                                            ['uclamplt', ['calldataload', 0], ['mload', 128]],  # noqa: E501
-                                            ['assert', ['call', 0, ['calldataload', 0], ['calldataload', 32], 0, 0, 0, 0]],  # noqa: E501
-                                            'stop'],
-                                    0]]]  # noqa: E128
-
-
-# compiled byte code
-PAYGAS_contract_bytecode = '0x61004f56604035f55074010000000000000000000000000000000000000000608052600035608051811061002e57600080fd5b5060006000600060006020356000356000f161004957600080fd5b005b61000461004f0361000460003961000461004f036000f3'  # noqa: E501
-
-
-# address where this contract will be deployed
-PAYGAS_contract_address = generate_CREATE2_contract_address(
-    b'',
-    decode_hex(PAYGAS_contract_bytecode)
-)
+CREATE2_contract = {
+    'lll_code': ['seq',
+                    ['return',  # noqa: E127
+                        0,
+                        ['lll',
+                            ['seq',
+                                ['mstore', 32, ['sload', 0]],
+                                ['mstore', 64, 118167469832824678325960435],
+                                ['clamp_nonzero', ['create2', ['mload', 32], ['mload', 32], 85, 11]],  # noqa: E501
+                                ['sstore', 0, ['add', ['mload', 32], 1]],
+                                'stop'],
+                        0]]],  # noqa: E128
+    # compiled byte code
+    'bytecode': b'0x610039566000546020526a61beef6020526002603ef3604052600b6055602051602051fe8061002957600080fd5b50600160205101600055005b61000461003903610004600039610004610039036000f3',  # noqa: E501
```

Comment:
Just confirm with you, are the bytecodes generated by the latest Vyper sharding branch?

Reply:
Ah, no, I don't think so. I will have them compiled using Vyper sharding branch.